media/js/mozorg/contribute/contribute-2015-ga.js

```diff
@@ -154,9 +164,12 @@ $(function () {
 
         $(this).off('submit');
 
+
         window.dataLayer.push({
-            event: 'contribute-submit',
-            newsletterSignup: newsletterstate
+            'event': 'contribute-signup-submit',
+            'contributeArea': $('#inquiry-form').data('contribute-area'),
```

Comment:
Since `$('#inquiry-form')` is being called multiple times, let's cache it in a variable at the beginning of the script like `var $inquiryForm = $('#inquiry-form');`.